packages/examples/src/model/server.ts

```diff
@@ -0,0 +1,17 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
Please move it to `language-server-runner.ts`. I don't see the need for this artificial separation.

Reply:
Please move to `server-commons.ts`. I overlooked the circular dependency. 
The model sub-folder is not needed now. I want to think about how to organize/move the common code out of the examples after this PR is merged.